examples/base-cra-typescript/README.md

```diff
@@ -1,6 +1,6 @@
 # Create React App example with TypeScript
 
-[MUI Base](https://mui.com/base/getting-started/overview/#main-content) is a library of unstyled React UI components which includes prebuilt components with production-ready functionality, along with low-level hooks for transferring that functionality to other components.
+[MUI Base](https://mui.com/base/getting-started/overview/) is a library of unstyled React UI components which includes prebuilt components with production-ready functionality, along with low-level hooks for transferring that functionality to other components.
```

Comment:
There shouldn't be an anchor link with `#main-content` here. It's coming from Algolia. I have taken note in https://www.notion.so/mui-org/Opportunities-408a9771556a4715830d18f3749af753#8a43dc8855e648d09102e9836709b4c1.